doc/changes.rst

```diff
@@ -49,6 +49,8 @@ Breaking Changes
     repo.get_views_traffic().views.timestamp
     repo.get_clones_traffic().clones.timestamp
 
+* Property ``AppAuth.private_key`` has been removed as part of (`#3065 <https://github.com/PyGithub/PyGithub/pull/3065>`_) (`36697b22 <https://github.com/PyGithub/PyGithub/commit/36697b22>`_)
```

Comment:
I think the first link doesn't need parentheses.
```suggestion
* Property ``AppAuth.private_key`` has been removed as part of `#3065 <https://github.com/PyGithub/PyGithub/pull/3065>`_ (`36697b22 <https://github.com/PyGithub/PyGithub/commit/36697b22>`_)
```

Reply:
```suggestion
* Property ``AppAuth.private_key`` has been removed (`#3065 <https://github.com/PyGithub/PyGithub/pull/3065>`_) (`36697b22 <https://github.com/PyGithub/PyGithub/commit/36697b22>`_)
```